src/main/services/logger.service.ts

```diff
@@ -0,0 +1,14 @@
+import {injectable} from "inversify";
+
+/**
+ * Logger class for beautiful logs <3
```

Comment:
This file probably belongs in a separate PR as well. Right now, it feels quite incomplete (maybe it should be using `chalk` under the hood so different types of messages appear in different colours?).